### DISCUSSION TITLE
Error in entry.worker.js

### DISCUSSION THREAD

**Discussion by WreewanMorhee** [ShafSpecs/remix-pwa#143] [Q&A]
I follow this tutorial to build my remix pwa: https://blog.logrocket.com/building-remix-pwa/
and I always found an error like image below
anyone knows why ??

<img width="767" alt="Screenshot 2024-03-19 at 3 02 53 PM" src="https://github.com/remix-pwa/remix-pwa/assets/33398071/76f54ee3-8f0d-4636-881e-171f50864f94">

here is all my remix version:
	"@remix-run/css-bundle": "^2.0.1",
    "@remix-run/node": "^2.0.1",
    "@remix-run/react": "^2.0.1",
    "@remix-run/serve": "^2.0.1",
    "@remix-pwa/cache": "^2.0.12",
    "@remix-pwa/sw": "^2.1.12",
    "@remix-pwa/strategy": "^2.1.9",
    "@remix-pwa/push": "^1.0.9",
    "@remix-pwa/client": "^2.0.1",
    "@remix-pwa/sync": "^2.0.1"

**Comment by ShafSpecs** [comment]
`@remix-pwa/cache` package has been deprecated. Remix PWA v4 introduced a lot of breaking changes, and the backwards compatibility between 3 and 4 is not there. Check out https://remix-pwa.run for the updated version and up-to-date guides
